File: core/support.go
package core

// Where a user ends up when nothing works. Every engine can fail at once —
// during a full shutdown they will — and an app that just says "failed" in
// that moment is useless. The channel is reachable over the same networks
// that stay up when everything else does not, and it is where fresh configs
// and news get posted.
const (
	SupportChannel = "https://t.me/parsv2r"
	SupportHandle  = "@parsv2r"
)

File: core/src/main/kotlin/xyz/jmc/gozar/core/Support.kt
package xyz.jmc.gozar.core

/**
 * Where a user ends up when nothing works.
 *
 * Every engine can fail at once — during a full shutdown they will — and an app
 * that answers that moment with "failed" is useless. This is the fallback the
 * UI points at, and where fresh configs and news get posted.
 */
object Support {
    const val CHANNEL_URL = "https://t.me/parsv2r"
    const val CHANNEL_HANDLE = "@parsv2r"
}
